src/verify_cert.rs

```diff
@@ -841,68 +841,23 @@ mod tests {
         intermediate_count: usize,
         trust_anchor_is_actual_issuer: TrustAnchorIsActualIssuer,
     ) -> Error {
-        use crate::{extract_trust_anchor, ECDSA_P256_SHA256};
-        use crate::{EndEntityCert, Time};
-
-        let alg = &rcgen::PKCS_ECDSA_P256_SHA256;
-
-        let make_issuer = || {
-            let mut ca_params = rcgen::CertificateParams::new(Vec::new());
-            ca_params
-                .distinguished_name
-                .push(rcgen::DnType::OrganizationName, "Bogus Subject");
-            ca_params.is_ca = rcgen::IsCa::Ca(rcgen::BasicConstraints::Unconstrained);
-            ca_params.key_usages = vec![
-                rcgen::KeyUsagePurpose::KeyCertSign,
-                rcgen::KeyUsagePurpose::DigitalSignature,
-                rcgen::KeyUsagePurpose::CrlSign,
-            ];
-            ca_params.alg = alg;
-            rcgen::Certificate::from_params(ca_params).unwrap()
-        };
-
-        let ca_cert = make_issuer();
+        let ca_cert = make_issuer("Bogus Subject".into());
```

Comment:
This is nice though commit messages with bullet lists are always a trigger for me -- I'd make this 3 commits.

Reply:
> commit messages with bullet lists are always a trigger for me

:laughing: I should have known better than to be lazy like that. Thanks for the prod. It's 3 commits now.